dot/peerset/peerset.go

```diff
@@ -340,8 +342,9 @@ func (ps *PeerSet) reportPeer(change ReputationChange, peers ...peer.ID) error {
 					return err
 				}
 
+				// TODO: change Noop to Drop once #2098 is fixed
```

Comment:
let me know if i should put commenting out this ban peer stuff into another PR

Reply:
Can you not just uncomment this out for now? I wasn't able to reproduce the nodes banning at this point at least locally.